src/components/BInput/BInput.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex">
+  <div class="flex column">
```

Comment:
We may want to change the class name from column to flex-column or something else.  Column is really generic.

Reply:
I like that idea. I think we do need a general naming convention for our classes. @stevenmwade suggested `b-*` and then most people didn't respond to the poll